### PR TITLE
Adding a clean sub command

### PR DESCRIPTION
**Pull request by cam156**

### CODE REVIEW

File: exe/solr_wrapper [hunk 1]
```diff
@@ -73,6 +93,18 @@ end
...
 instance = SolrWrapper.instance(options)
+
+if command == 'clean'
+  if instance.started?
+    $stderr.print "Please stop soll before cleaning\n"
+    exit 1
+  end
+  $stderr.print "cleaning #{instance.instance_dir}...\n"
+  instance.remove_instance_dir!
+  instance.extract_and_configure
```
Comment (cbeer) [inline exe/solr_wrapper]: Do we need `extract_and_configure`? here?

Comment (cam156) [inline exe/solr_wrapper]: I was copying the clean from the rake task.  I'm willing to not do it...

Comment (cbeer) [inline exe/solr_wrapper]: Yeah, I don't see any reason to immediately extract solr again, so we can just be lazy.

File: exe/solr_wrapper [hunk 2]
```diff
@@ -73,6 +93,17 @@ end
 options[:verbose] = true if options[:verbose].nil?
 
 instance = SolrWrapper.instance(options)
+
+if command == 'clean'
+  if instance.started?
+    $stderr.print "Please stop soll before cleaning\n"
```
Comment (cbeer) [inline exe/solr_wrapper]: `s/soll/solr/`, please.

And, can we just make these `.puts` without the trailing `\n`?

Comment (cam156) [inline exe/solr_wrapper]: 👌

**Pull request merged**